index.js

```diff
@@ -76,19 +74,22 @@ client.on('guildMemberAdd', member => {
 // ================ On message ================
 // Handler for responding to messages (a la slackbot).
 
-client.on('message', message => {
+client.on('message', (message) => {
   // Only respond to messages sent from real users and those that are
   // prefixed appropriatly.
   if (!message.content.startsWith(prefix) ||
-      message.author.bot ||
-      message.channel.name !== 'zerowatch-bot-testing') return;
+      message.author.bot) return;
 
-  const args = message.content.slice(prefix.length).split(/ +/);
+  // Temporary, limit commands to a single channel.
+  if (message.channel.name !== 'general') return;
```

Comment:
Are you using the general channel for testing and not the zerowatch-bot-testing channel?